sdk/aqueduct/client.py

```diff
@@ -178,7 +178,15 @@ def github(self, repo: str, branch: str = "") -> Github:
         """
         return Github(repo_url=repo, branch=branch)
 
-    def create_param(self, name: str, default: Any, description: str = "") -> BaseArtifact:
+    def create_param(
+        self,
+        name: str,
+        default: Any,
+        description: str = "",
+        use_local: bool = False,
+        as_type: Optional[ArtifactType] = None,
```

Comment:
Can we just call this `as`?

Reply:
I think `as` is a python keyword, overriding it with a variable might be a bit tricky. Will something like `_as` or `astype` suffice?